### PR TITLE
2020-01-02

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2020-01-02-streaming-cable.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+status: publish
+categories: [Tech]
+image:
+image2:
+author: Valentino Urbano
+---
+
+Streaming platforms were born as the final answer to cable. The solution to the problem of having 100s of channel and having to pay different rates for each of them. We will have one streaming platform with everything and pay one single fee.
+
+It did not pan out as planned. We now have more than 10 streaming subscriptions each one of them with their own original shows and exclusive publishing deals for specific movies and tv shows and we ended up back where we started. If someone wants to watch some specific content he will have to subscribe to multiple streaming channels and pay multiple subscribptions. This will non duoubtly bring back piracy that seemed to have been killed by the fair pricing on streaming services. Spending 9.99 a month is not a big deal, spending 3 or 4 times as much to watch all the shows you care about is a different story.
```
Comment (MsSlacker) [inline _posts/2020-01-02-streaming-cable.md]: subscribptions - > subscriptions

Comment (MsSlacker) [reply _posts/2020-01-02-streaming-cable.md]: non duoubtly - > no doubt

File: _posts/2020-01-02-streaming-cable.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Streaming
+
+As an example these are the shows I'd be interested in and the respective platforms:
+
+- Game of Thrones (now finished) on HBO Go
+- Chernobyl (now finished) on HBO Go
+- The expanse on Netflix
```
Comment (MsSlacker) [inline _posts/2020-01-02-streaming-cable.md]: AMAZON PRIME

Comment (MsSlacker) [reply _posts/2020-01-02-streaming-cable.md]: Plus capitalize it lol

Comment (MsSlacker) [reply _posts/2020-01-02-streaming-cable.md]: We don't really watch anything on Netflix besides black mirror

File: _posts/2020-01-02-streaming-cable.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Pricing for one month:
+
+Amazon Prime Video $8.99
+HBO Go $14.99
+Netflix $12.99
+Apple TV Plus $4.99
+Disney Plus $6.99
+
+**Total $48.95** [^1]
```
Comment (MsSlacker) [inline _posts/2020-01-02-streaming-cable.md:42]: Want Polish prices?

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2020-01-02-streaming-cable.md (hunk 3 above)
Comment (valeIT) [reply _posts/2020-01-02-streaming-cable.md:42]: not really xD just an average

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**